charts/ceph-csi-cephfs/README.md

```diff
@@ -50,6 +50,34 @@ After installation succeeds, you can get a status of Chart
 helm status "ceph-csi-cephfs"
 ```
 
+### Upgrade Chart
+
+If you want to upgrade your Chart, use the following commands.
+```bash
+helm repo update ceph-csi
+helm upgrade --namespace ceph-csi-cephfs ceph-csi-cephfs ceph-csi/ceph-csi-cephfs
+```
+
+For upgrading to a specific version, provide the flag `--version` and the
+version.
+
+**Do not forget to include your values**, if they differ from the default values.
+We recommend not to use `--reuse-values` in case there are new defaults AND
+compare your currently used values with the new default values.
+
+#### Known Issues Upgrading
+
+- When upgrading to version >=3.7.0, you might encounter an error that the
+  CephFS CSI Driver cannot be updated (https://github.com/ceph/ceph-csi/issues/3397).
```

Comment:
```suggestion
CephFS CSI Driver cannot be updated. Please refer to [issue](https://github.com/ceph/ceph-csi/issues/3397) for more details.
```

Reply:
I also used a bare URL, which I need to fix.

Reply:
I assue the dedent was an accident?

Reply:
Yes correct , suggestion is to use link instead of just issue URL

Reply:
Rerunning `make containerized-test TARGET=lint-extras` before pushing this time… – hopefully for the last time :smile:

Reply:
`2 chart(s) linted, 0 chart(s) failed` :tada:

Reply:
Thanks for testing it locally :100:

Reply:
I could have avoided half the comments, if I did in the first place